tools/wptrunner/wptrunner/browsers/chrome.py

```diff
@@ -51,7 +53,15 @@ def executor_kwargs(test_type, server_config, cache_manager, run_info_data,
         capabilities["chromeOptions"]["excludeSwitches"] = ["enable-automation"]
     if test_type == "wdspec":
         capabilities["chromeOptions"]["w3c"] = True
-    executor_kwargs["capabilities"] = capabilities
+
+    if __wptrunner__["executor"]["testharness"] == "WebDriverTestharnessExecutor" \
+        or __wptrunner__["executor"]["reftest"] == "WebDriverRefTestExecutor":
```

Comment:
This isn't really a swappable thing, so I wonder what the intent here is? Per the current design you would have to make this a different product like `chrome-webdriver`.

Reply:
I didn't consider that for some reason... that sounds like a cleaner way to do it. Good suggestion :)